Tests/GRPCTests/ConnectionManagerTests.swift

```diff
@@ -1169,16 +1210,26 @@ extension ConnectionManagerTests {
 
     let http2 = HTTP2Delegate()
 
+    var streamDelegate: NIOHTTP2StreamDelegate?
```

Comment:
We shouldn't escape the delegate from the channel provider closure here.

Instead, when we need it later on, we can grab the `GRPCIdleHandler` from the channel and call it directly.